doc/devel/coding_guide.rst

```diff
@@ -63,7 +63,18 @@ PR Review guidelines
   :file:`doc/api/api_changes` and significant new features have and
   entry in :file:`doc/user/whats_new`.
 
-* Make sure the Travis, Appvyor, circle, and codecov tests are passing
+  - If a PR already has a positive review, a core developer (e.g. the first
+    reviewer, but not necessarily) may champion that PR for merging.  In order
+    to do so, they should ping all core devs both on github and on the dev
+    mailing list.  Other core devs can then either review the PR and merge or
+    reject it, or simply request that it gets a second review before being
+    merged.  If no one requires such a second review within a week, the PR can
```

Comment:
```suggestion
    merged.  If no one provides such a second review within a week, the PR can
```

Reply:
actually the point is "if no one says 'I think this needs a second review, even though I cannot provide it myself'"

Reply:
Suggest we tag these so folks can easily find them?

Reply:
sure.  you get to pick the color of the "Merge with single review?" label :)

Reply:
Green like "Needs Review"?

Reply:
deal, label created and PR edited